melos.yaml

```diff
@@ -0,0 +1,9 @@
+name: snapd
+
+packages:
+  - .
```

Comment:
I've had a chat with @spydon about this - using the current directory as a joint place for the melos workspace and the dart package can potentially lead to some annoying dependency issues, since `melos` becomes a dev-dependency of the actual package. We could avoid this by moving the package code into a dedicated `packages` directory (even if we only maintain a single package here).